src/momento/internal/_utilities/_grpc_channel_options.py

```diff
@@ -1,14 +1,17 @@
 from __future__ import annotations
 
-import grpc
+from typing import Sequence, Tuple, Union
 
 from momento.config.transport.grpc_configuration import GrpcConfiguration
+from momento.config.transport.transport_strategy import StaticGrpcConfiguration
 from momento.internal._utilities import _timedelta_to_ms
 
 DEFAULT_MAX_MESSAGE_SIZE = 5_243_000  # bytes
 
+ChannelArguments = Sequence[Tuple[str, Union[int, None]]]
```

Comment:
After refactoring into two methods, the linter complained:

> `src/momento/internal/_utilities/_grpc_channel_options.py:69:12: error: Expression has type "Any"`

So I made a type alias with `Any` in it that should be accepted by both async and sync versions

Reply:
This was after looking up that the grpc.aio return type was `(type alias) ChannelArgumentType: type[Sequence[Tuple[str, Any]]]`

Reply:
Thanks for doing that. The Python gRPC library doesn't have type hints so we typically just add linter ignores to them in `pyproject.toml`. This is good too though!